src/Microsoft.Health.Fhir.Shared.Api/Controllers/ConvertDataController.cs

```diff
@@ -28,7 +28,6 @@ namespace Microsoft.Health.Fhir.Api.Controllers
 {
     [ServiceFilter(typeof(AuditLoggingFilterAttribute))]
     [ServiceFilter(typeof(OperationOutcomeExceptionFilterAttribute))]
-    [ServiceFilter(typeof(ValidateContentTypeFilterAttribute))]
```

Comment:
Did this need the _format or _accept header validation?

Reply:
No matter what you pass to convert it's always return you "text/plain"
```

            return new ContentResult
            {
                Content = response.Resource,
                ContentType = "text/plain",
            };
```

---
In reply to: [587696839](https://github.com/microsoft/fhir-server/pull/1699#discussion_r587696839) [](ancestors = 587696839)

Reply:
I personally don't see much reason to validate something we gonna ignore anyway, why waste CPU cycles?

---
In reply to: [587701608](https://github.com/microsoft/fhir-server/pull/1699#discussion_r587701608) [](ancestors = 587701608,587696839)